jwriter/package_example_test.go

```diff
@@ -0,0 +1,19 @@
+package jwriter
+
+import "fmt"
+
+func Example() {
```

Comment:
I'm not sure exactly what this file is for?

Reply:
https://blog.golang.org/examples

Reply:
TIL